### PR TITLE
added logging to handlers

### PR DESCRIPTION
**Pull request by kimbellG**

### CODE REVIEW

File: core/handler/tournament-handler.go [hunk 1]
```diff
@@ -8,15 +8,25 @@ import (
...
+	log "github.com/sirupsen/logrus"
 	"google.golang.org/protobuf/types/known/emptypb"
 )
 
 func (sh *ServiceHandler) CreateTournament(ctx context.Context, r *ttgrpc.CreateTournamentRequest) (*ttgrpc.CreateTournamentResponse, error) {
+	createLog := log.WithFields(log.Fields{
+		"action":  "create tournament",
+		"request": r,
+	})
+	createLog.Info("received create tournament request")
```
Comment (khepsib) [inline core/handler/tournament-handler.go:20]: log.WithFields(log.Fields{
	"action":  "create tournament",
	"request": 
}).Info("received create tournament request")

File: core/handler/tournament-handler.go [hunk 2]
```diff
@@ -8,15 +8,25 @@ import (
...
+
 	id, err := sh.tournamentController.Create(ctx, tournamentFromProto(r))
 	if err != nil {
+		kerror.ErrorLog(createLog, err, "failed to create tournament")
```
Comment (khepsib) [inline core/handler/tournament-handler.go:24]: I'm not sure why you log in kerror package. Can you explain?

I would rather just call something like logrus.WithFields(logrus.Fields{}).Error("bla bla")

File: core/handler/tournament-handler.go [hunk 3]
```diff
@@ -30,15 +40,24 @@ func tournamentFromProto(protoTournament *ttgrpc.CreateTournamentRequest) *model
 }
 
 func (sh *ServiceHandler) GetTournamentByID(ctx context.Context, r *ttgrpc.TournamentRequest) (*ttgrpc.Tournament, error) {
+	getlog := log.WithFields(log.Fields{
+		"action":  "get tournement by id",
+		"request": r,
+	})
+	getlog.Info("received get touranment by id request")
```
Comment (khepsib) [inline core/handler/tournament-handler.go:47]: the same here

File: core/handler/tournament-handler.go [hunk 4]
```diff
@@ -66,45 +85,75 @@ func uuidOfUsersToStringSlice(users []models.User) []string {
 }
 
 func (sh *ServiceHandler) JoinTournament(ctx context.Context, r *ttgrpc.JoinRequest) (*emptypb.Empty, error) {
+	joinlog := log.WithFields(log.Fields{
+		"action":  "join to tournament",
+		"request": r,
+	})
+	joinlog.Info("received join touranament request")
```
Comment (khepsib) [inline core/handler/tournament-handler.go:92]: and here

File: core/handler/tournament-handler.go [hunk 5]
```diff
@@ -66,45 +85,75 @@ func uuidOfUsersToStringSlice(users []models.User) []string {
...
+
 	return &emptypb.Empty{}, nil
 }
 
 func (sh *ServiceHandler) FinishTournament(ctx context.Context, r *ttgrpc.TournamentRequest) (*emptypb.Empty, error) {
+	finishlog := log.WithFields(log.Fields{
+		"action":  "finish tournament",
+		"request": r,
+	})
+	finishlog.Info("received finish tournament request")
```
Comment (khepsib) [inline core/handler/tournament-handler.go:121]: and here

File: core/handler/tournament-handler.go (hunk 1 above)
Comment (kimbellG) [reply core/handler/tournament-handler.go:20]: i create logrus.Entry and then put this to ErrorLog function, also this require for eliminate duplicates when i'm logging result of request

**Pull request merged**